cilium-sysdump/__main__.py

```diff
@@ -77,12 +86,12 @@ def parse_comma_sep_list(arg_string):
                         help='only return logs newer than a '
                              'relative duration like 5s, 2m, or'
                              ' 3h. Defaults to 30m.',
-                        default='30m')
+                        default='0')
```

Comment:
The README should also be updated as all options (and their default values) are reported there as well

Reply:
Thanks!